lib/simplecov/exit_codes/minimum_coverage_by_file_check.rb

```diff
@@ -29,9 +31,30 @@ def exit_code
 
       attr_reader :result, :minimum_coverage_by_file
 
-      def covered_percentages
-        @covered_percentages ||=
-          result.covered_percentages.map { |percentage| SimpleCov.round_coverage(percentage) }
+      def coverage_statistics_by_file
+        @coverage_statistics_by_file ||=
+          (res = result.coverage_statistics_by_file).each do |criteria, stats|
+            res[criteria] = stats.map { |stat| SimpleCov.round_coverage(stat.percent) }
```

Comment:
Do we need this code?

I'm probably be wrong but we're "just" rounding the percentages, couldn't we do that down in `compute_minimum_violations` while we're mapping and call `SimpleCov.round_coverage` on `actual_percent` there? :thinking: